nixos/tests/akkoma.nix

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
Consider leaving a TODO to test that federation works (between two instances started in the test).

Reply:
Will do.